catalog/dags/providers/provider_api_scripts/rawpixel.py

```diff
@@ -80,6 +80,7 @@ class RawpixelDataIngester(ProviderDataIngester):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.api_key: str = Variable.get("API_KEY_RAWPIXEL")
+        self.headers = {"User-Agent": prov.UA_STRING}
```

Comment:
Not necessarily something we need to do in this PR, but it seems like we should just have this in the headers _by default_ for all our providers in order to be good citizens of the web. What do you think Staci? We can make a separate issue for that (since we overwrite `headers` in some cases which would blow away whatever default the base class provides so it becomes a bit more complicated).

Reply:
Sure, why not! Should be easily possible with the base class, just as long as we're careful to support extra headers. I'll make an issue :)

Reply:
#3683 created!